### PR TITLE
Azure build jobs for master.

### PR DESCRIPTION
**Pull request by kwoodson**
Adding PR jobs for CI.
- [x] azure_build_node_image_centos.yml
- [x] origin_release_install_azure.yml
- [x] test_branch_origin_extended_conformance_azure.yml
- [x] test_pull_request_origin_extended_conformance_azure.yml

### CODE REVIEW

File: sjb/config/test_cases/azure_build_node_image_centos.yml [hunk 1]
```diff
@@ -0,0 +1,60 @@
+---
+provision:
+  os: "rhel"
+  stage: "base"
+  provider: "aws"
+sync:
+  - openshift,openshift-ansible=release-3.9
```
[the remark this answers is not in the document]
Comment (kwoodson) [reply sjb/config/test_cases/azure_build_node_image_centos.yml]: Thanks, fixed.

**Comment by kargakis** [comment]
Jobs pushed in Jenkins

File: sjb/config/common/test_cases/origin_release_install_azure.yml [hunk 2]
```diff
@@ -0,0 +1,200 @@
...
+    - name: "ADDITIONAL_SKIP"
+      description: |-
+        Regular expression to filter additional tests from the conformance suite.
+        See also:
+        <div>
+        </div>
+    - name: "OS_PUSH_BASE_REGISTRY"
+      default_value: "registry.svc.ci.openshift.org/"
+    - name: "OS_PUSH_BASE_REPO"
+      default_value: "ci-pr-images/"
```
Comment (kargakis) [inline sjb/config/common/test_cases/origin_release_install_azure.yml]: @smarterclayton how do we prune these today? [AFAIK `image-pruner` does prune only old tags](https://github.com/openshift/release/blob/f896f1bece92cca7b74dfbd783fec8ec63a050f5/cluster/ci/jobs/image-pruner.yaml#L33).

Comment (jim-minter) [reply sjb/config/common/test_cases/origin_release_install_azure.yml]: that's not answered by the `oc get is -o name -n ci-pr-images | grep "${INSTANCE_PREFIX}" | xargs -r oc delete` below?

Comment (kargakis) [reply sjb/config/common/test_cases/origin_release_install_azure.yml]: Yes, it does, thanks

**Pull request merged**